std/internal/math/biguintcore.d

```diff
@@ -344,6 +344,17 @@ public:
         return buff[z-frontExtraBytes..$];
     }
 
+    /**
+     * Convert to an octal string.
+     */
+    char[] toOctalString() const
+    {
+        auto predictLength = 8*BigDigitBits / 3;
```

Comment:
An assertion may be added to check that exactly 0, 1, or 2 characters of the predicted length are wasted.